releasenotes/notes/add-clifford-t-transpiler-pipeline-e640b5ba64eab13a.yaml

```diff
@@ -0,0 +1,41 @@
+---
+features_transpiler:
+  - |
+    The function :func:`.generate_preset_pass_manager` now generates a special
+    pass manager when the basis set consists of Clifford+T gates only.
+    Formally, a Clifford+T basis set must contain either a :class:`.TGate` or
+    :class:`.TdgGate` (or both), and only Clifford gates in addition.
+    The full list of supported Clifford gates can be obtained using
+    :func:`.get_clifford_gate_names`.
+
+    For example::
+  
+      from qiskit.circuit import QuantumCircuit
+      from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+      from qiskit.quantum_info import get_clifford_gate_names
+
+      basis_gates = get_clifford_gate_names() + ["t", "tdg"]
+      pm = generate_preset_pass_manager(basis_gates=basis_gates)
+
+      qc = QuantumCircuit(1)
```

Comment:
perhaps it's worth to provide a circuit with 2-3 qubits? otherwise, the user could just use SK plugin

Reply:
There are more examples available as tests. 😄

Reply:
I know that there are tests, still the user would like to know what's new in here compared to using SK directly, and I think that essentially it's the fact that you can do it for several qubits.

Reply:
Sure, added a slightly larger example in [a6799f1](https://github.com/Qiskit/qiskit/pull/14225/commits/a6799f1d8e0966afbc5cc69734b5c79391cd312c).